cipher.py

```diff
@@ -6,18 +6,6 @@
 logs = []
```

Comment:
Move log into another class. Such that you can add to the log from anywhere else.